docs/units/standard_units.rst

```diff
@@ -48,6 +48,56 @@ all the existing predefined units of a given type::
     u            | 1.66054e-27 kg  | Da, Dalton                       ,
   ]
 
+.. _doc_unit_prefixes:
+
+The unit prefixes
+----------------------
+
+In addition to these units, `astropy.units` includes the concept of unit prefixes. A unit prefix is a specifier or mnemonic that is prepended to units of measurement to indicate multiples or fractions of the units. Units of various sizes are commonly formed by the use of such prefixes.
+
+SI Prefixes
+^^^^^^^^^^^
+The prefixes of the metric system (commonly known as SI prefixes) precede a basic unit of measure to indicate a decadic multiple (base 10) and fraction of a unit. The following are supported::
+
+
+    Symbol | Unit name  | Value
```

Comment:
Instead of `::` block quote, it would be nicer to have a properly formatted table.